appcues/src/main/java/com/appcues/trait/AppcuesTraitFactories.kt

```diff
@@ -55,6 +47,14 @@ internal fun Scope.backgroundContentTraitFactory(): TraitFactoryBlock {
     return { config, level, _ -> get<BackgroundContentTrait> { parametersOf(config, level) } }
 }
 
+internal fun Scope.modalTraitFactory(): TraitFactoryBlock {
```

Comment:
all presenting traits needs to know whats the renderContext ( this was an oversight on my last pass, where i focused so  much more on Actions and did not think too much about traits)